Readme.md

```diff
@@ -1,89 +1,95 @@
 # ezl: easyLambda
-> Parallel dataflows for easy data processing with functional, C++ and MPI
+> Parallel data processing made easy using dataflows in modern C++
 
-## Long story short
+## Design Goals
```

Comment:
how about borrowing [What is easyLambda](https://haptork.github.io/easyLambda#what-is-easyLambda) section from the webpage (after possible modifications).